### PR TITLE
0 errors this time

### PR DESCRIPTION
**Pull request by fzipi**
Looks like facebook can't be reached for two participants, maybe some typo from them?

This pull solves all problems until today. We should go for the next step!

### CODE REVIEW

File: src/includes/blocks/technology.pug [hunk 1]
```diff
@@ -3,7 +3,7 @@
     .container
         h1 {{ page.title }}  (Technology)
         p {{ page.description }}
-        p {{ content }}
+        {{ content }}
```
Comment (DinisCruz) [inline src/includes/blocks/technology.pug:6]: Doesn't this change break Jade's build?

Comment (fzipi) [reply src/includes/blocks/technology.pug:6]: It didn't.

That p was adding an extra empty p to the generated content form markdown. You can see an example with it in file './_site/Working-Sessions/Technologies/GDPR.html' line 54-55.

That's what htmlproofer didn't like.

Comment (DinisCruz) [reply src/includes/blocks/technology.pug:6]: weird that jade doesn't pick it up

Webstorm did

image

Comment (DinisCruz) [reply src/includes/blocks/technology.pug:6]: here is a better way to do it
image
which creates this html

image

which looks like created the correct html

image

**Pull request merged**